oca_port/squash_bot_commits.py

```diff
@@ -0,0 +1,154 @@
+import subprocess
+from .utils.misc import Output, bcolors as bc
+import click
+import tempfile
+import os
+from .utils import git as g
+
+
+MESSAGE_TO_SQUASH = [
+    "Added translation using Weblate",
+    "Translated using Weblate",
+    "Update translation files",
+]
+AUTHOR_EMAILS_TO_SQUASH = [
+    "transbot@odoo-community.org",
+    "noreply@weblate.org",
+    "oca-git-bot@odoo-community.org",
+    "oca+oca-travis@odoo-community.org",
+    "oca-ci@odoo-community.org",
+    "shopinvader-git-bot@shopinvader.com",
+]
+
+
+class SquashCommitBot(Output):
```

Comment:
```suggestion
class SquashBotCommits(Output):
```

Reply:
or is it a Bot that squashes bot commits? :)

Reply:
it's funny that I came up with the exact same name but then named differently as I think it behaves like a Bot :rofl: 

Go with SquashBotCommits then